src/components/views/settings/shared/SettingsSubsection.tsx

```diff
@@ -22,6 +23,8 @@ export interface SettingsSubsectionProps extends HTMLAttributes<HTMLDivElement>
     heading: string | React.ReactNode;
     description?: string | React.ReactNode;
     children?: React.ReactNode;
+    // when true content will be justify-items: stretch
+    stretchContent?: boolean;
```

Comment:
also: is it actually used?

Reply:
It applies the class `mx_SettingsSubsection_contentStretch`, which sets a different justification for the contents. It's used to stretch the content to full width where necessary, eg with fields.
Strictly shouldn't be in this PR after the split, it's [used downstream](https://github.com/matrix-org/matrix-react-sdk/blob/1484cb20a5f390dd7f9c0ce2e7a38616d23b7e98/src/components/views/settings/tabs/user/VoiceUserSettingsTab.tsx#L157).
I don't think it's worth pulling out because these PRs are going to merged in quick succession.

Reply:
> It applies the class `mx_SettingsSubsection_contentStretch`, which sets a different justification for the contents. It's used to stretch the content to full width where necessary, eg with fields.

could you add a comment to this effect to help readers like me who are less familiar with slightly unusual CSS properties like this?

```suggestion
    // when true content will be justify-items: stretch, which will make items within the section stretch to full width.
    stretchContent?: boolean;
```

Reply:
> Strictly shouldn't be in this PR after the split, it's [used downstream](https://github.com/matrix-org/matrix-react-sdk/blob/1484cb20a5f390dd7f9c0ce2e7a38616d23b7e98/src/components/views/settings/tabs/user/VoiceUserSettingsTab.tsx#L157).
> I don't think it's worth pulling out because these PRs are going to merged in quick succession.

That's fair, but I think it's worth noting that stuff like this makes it harder to review a PR, because it's hard to see how the changes within the PR fit together. In future if you could leave a comment on the PR explaining, that would be really helpful.